traffic_ops/testing/api/v4/cdn_locks_test.go

```diff
@@ -487,6 +555,33 @@ func TestCDNLocks(t *testing.T) {
 								}
 							})
 						}
+					case "STATIC DNS ENTRIES POST":
+						{
+							t.Run(name, func(t *testing.T) {
+								alerts, reqInf, err := testCase.ClientSession.CreateStaticDNSEntry(staticDNSEntry, testCase.RequestOpts)
+								for _, check := range testCase.Expectations {
+									check(t, reqInf, nil, alerts, err)
+								}
+							})
+						}
+					case "STATIC DNS ENTRIES PUT":
+						{
+							t.Run(name, func(t *testing.T) {
+								alerts, reqInf, err := testCase.ClientSession.UpdateStaticDNSEntry(testCase.EndpointId(), staticDNSEntry, testCase.RequestOpts)
+								for _, check := range testCase.Expectations {
+									check(t, reqInf, nil, alerts, err)
+								}
+							})
+						}
+					case "STATIC DNS ENTRIES DELETE":
```

Comment:
nit: would it make sense to include `default` switch cases that fail the test? That way we are less prone to typos sneaking in and causing the tests to not actually be run.